content/spin/install.md

```diff
@@ -22,7 +22,11 @@ Spin runs on Linux (amd64 and arm64), macOS (Intel and Apple Silicon), and Windo
 
 {{ startTab "Linux"}}
 
-There are multiple ways to install Spin. The easiest is to use the installer script, hosted on this site. The script installs Spin along with a starter set of language templates and plugins:
+You can now manage your Spin installation via [Homebrew](https://brew.sh/). Homebrew automatically installs Spin templates and Spin plugins, and on uninstall, will prompt you to delete the directory where the templates and plugins were downloaded:
```

Comment:
We should say what that directory is (where plugins and templates are installed), namely `$HOMEBREW_PREFIX/fermyon-spin`

Reply:
Agree though this would be best done i https://developer.fermyon.com/spin/cache rather than here